ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installation-nightly-4.21.yaml

```diff
@@ -61,6 +65,55 @@ resources:
       cpu: 400m
       memory: 800Mi
 tests:
+- as: aws-ipi-all-regions-f1
+  cron: 5 22 * * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_OBJECT: Regions
+    workflow: cucushift-installer-rehearse-aws-cases-clusters
+- as: aws-ipi-all-localzones-f1
+  cron: 13 18 * * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_OBJECT: LocalZones
+    workflow: cucushift-installer-rehearse-aws-cases-clusters
+- as: aws-ipi-all-wavelength-zones-f1
+  cron: 25 17 * * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_OBJECT: WavelengthZones
+    workflow: cucushift-installer-rehearse-aws-cases-clusters
+- as: aws-ipi-all-instance-types-f1
+  cron: 8 22 * * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_FILTERS: "57148"
+      TEST_OBJECT: InstanceTypes
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    - ref: openshift-extended-test
+    - ref: openshift-e2e-test-qe-report
+    workflow: cucushift-installer-rehearse-aws-cases-clusters
 - as: aws-ipi-all-local-zones-f60
```

Comment:
Once this PR get merged, seems that existing aws jobs for all region and all instance types can be removed?